app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,8 @@
   </head>
 
   <body>
+	<p class="notice"><%= notice %></p>
```

Comment:
`application.html.erb` is going to probably be the layout for the public site. The flash message should only be needed in the layout for the internal site (which we don't have yet). I think we can remove those flash messages for now.